lib/seam/http_multi_workspace.rb

```diff
@@ -29,29 +30,24 @@ def workspaces
         @workspaces ||= WorkspacesProxy.new(Seam::Clients::Workspaces.new(self))
       end
 
-      def self.from_personal_access_token(personal_access_token, endpoint: nil, wait_for_action_attempt: true)
+      def self.from_personal_access_token(personal_access_token, endpoint: nil, wait_for_action_attempt: true, client_options: {}, retry_options: {})
         new(
           personal_access_token: personal_access_token,
           endpoint: endpoint,
-          wait_for_action_attempt: wait_for_action_attempt
+          wait_for_action_attempt: wait_for_action_attempt,
+          client_options: client_options,
+          retry_options: retry_options
         )
       end
 
       def request_seam_object(method, path, klass, inner_object, config = {})
-        response = request_seam(method, path, config)
-
-        data = response[inner_object]
-
+        response = Seam::Http::Request.request_seam(@client, @endpoint, method, path, config)
```

Comment:
Not in scope for this PR, but we should follow a transparent pattern for using the client as was done for Python: https://github.com/seamapi/python/blob/cb26ee4f9544fafae3e3614f1b02eda41a3445a6/seam/routes/access_codes_simulate.py#L23-L27